src/content/development/contribution-guide/_index.en.md

```diff
@@ -222,15 +224,32 @@ To be expanded
 
 ### Unplanned Work
 
-To be expanded
+During the work on a release, there will be tasks which the team have to add to the projects which was unknown at the planning phase.
+There are 3 type of those tasks:
 
-#### Ongoing Maintenance
+* [Bugs](#bugs).
+* Ongoing maintenance [tasks](#tasks).
+* New [epics](#epics) or independent [tasks](#tasks).
 
-To be expanded
+The team triages all tasks and bugs as part of a weekly meeting.
+The bugs can be worked on immediately, while other types of issues are described below.
 
 #### Exception Process
 
-To be expanded
+In some cases, a task or an epic will be reported in the middle of a release cycle.
+Small and medium sized tasks will be triaged and prioritized in the same manner as other issues.
+Once triaged and prioritized, small and medium tasks can be worked on according to their priority.
+
+Epics would need to undergo an exception process.
+The epic's proponent would need to bring it up to discussion in the weekly community meeting.
+It's up to them to convince why this issue takes precedence over other pre-planned and triaged issues.
+A 50% approval from the project team is required for it to be accepted in the middle of the release.
```

Comment:
```suggestion
A 50% approval from the Project Owners is required for it to be accepted in the middle of the release.
```

Reply:
What is the "project team"? I don't think that defines any clear group of people. Also, the Project Owners are the ones that ultimately choose the technical direction of the project and guide releases. Please see https://submariner.io/community/contributor-roles/.

Reply:
@mkolesnik - since this phrase was your suggestion, do you think we should change this to `Project Owners`?
@dfarrell07 - the approval process should be done during the weekly community meeting, hence `project team` is considered as the meeting participants

Reply:
We can change to "project members" if you want to reflect the community aspect.

I also think "project committers" can be fine if we want a stricter list.

IMHO "project owners" is too narrow as it only has 3 people in actuality, and I dont see why committers cant cast a vote in these decisions.

Reply:
I think "Project Committers" is fine as it's a well-defined group and makes sense in practice. Or maybe "Committers to the files expected to be impacted".

But to be clear for the record, this is actually delegating a part of a responsibility that's ultimately granted to Project Owners by Submariner's governance.

Owner Responsibilities and Privileges:
* Make and approve technical design decisions for the project
* Set technical direction and priorities for the project
* Define milestones and releases

https://submariner.io/community/contributor-roles/#owner-responsibilities-and-privileges

Of course it's fine and normal to delegate responsibilities to spread out work to the most relevant stakeholders. Just want to be clear that they are already defined as the ultimate authority in this domain.

Reply:
I'm also okay with "Project Members", but I'm not sure it would be a good use of the time of that large group of lightly-connected-to-SubM people. I think we'd have low voter turnout or have to spam folks to ask them to spend time on something that they likely don't really care about.

Reply:
```suggestion
A 50% approval from the [Project Members](https://submariner.io/community/contributor-roles/#member) in attendance at the meeting or having voted before the meeting is required for an epic to be accepted in the middle of the release.
```